src/tokens/utils.ts

```diff
@@ -33,7 +33,11 @@ export function sortByUsdBalance(token1: TokenBalance, token2: TokenBalance) {
 }
 
 export function isStablecoin(token: TokenBalance | undefined) {
-  return token?.isCoreToken && token.symbol !== 'CELO'
+  return (
+    token?.isCoreToken &&
+    token.symbol !== 'CELO' &&
+    token.networkId === networkConfig.networkToNetworkId[Network.Celo]
+  )
```

Comment:
there is an `isStableCoin` property now in address-metadata, can we just use that?

Reply:
oh interesting, this function is used to see if it's a contract that supports transfer comments. So misleading! That's not a property of stablecoins, it's a property of a subset of Celo contracts...the function should be renamed to make that more clear

Reply:
Yeah -- it's pretty weird. I've updated the function name to reflect this usage.